src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -114,6 +115,13 @@
  */
 public interface CLinker {
 
+    /**
+     * The value returned by the process when the target of an upcall throws an exception.
+     *
+     * @see CLinker#upcallStub(MethodHandle, FunctionDescriptor, ResourceScope)
+     */
+    int ERR_UNCAUGHT_EXCEPTION = privilegedGetProperty("jdk.incubator.foreign.uncaught_exception_code", 1);
```

Comment:
Do we need a system property? E.g. can it be a constant value?

Reply:
We could just use `1` here as well, but during testing I've found that the default uncaught exception handler also return `1`, so I was getting a false positive.

Reply:
Thinking some more. Having a system property is probably overkill, and checking stderr for the exception message in the test is good enough